index.html

```diff
@@ -6,11 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header class="nav">
+        <nav class="nav-bar">
```

Comment:
When you validated this HTML, what result did you get?

What is the meaning of `<ul>` ? Is there another piece to this code that might help you here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Reply:
I have changed to <a> think this is better?